tools/validators/instance_validator/tests/telemetry_validator_test.py

```diff
@@ -43,6 +44,10 @@
                                        'message_attributes_SDC_EXT-17.json')
 _MESSAGE_ATTRIBUTES_3 = json.load(open(_MESSAGE_ATTRIBUTES_PATH_3))
 
+_MESSAGE_ATTRIBUTES_PATH_FACILITIES = \
```

Comment:
revert all unnecessary explicit linebreaks.

Reply:
what do you think about bringing all in 3p then `g4 fix` then update to github?

Reply:
updated the explicit linebreaks